Improve error message from `ParamDMD` when `parameters` isn't set

The current message printed when `ParamDMD.parameters` is not set may not not be informative enough for newcomers.


```python
pydmd/paramdmd.py:544: in _interpolate_missing_modal_coefficients
    list(
pydmd/paramdmd.py:541: in interpolate_future_pod_coefficients
    return approx.predict(self.parameters)
/usr/local/lib/python3.9/site-packages/ezyrb/rbf.py:63: in predict
    return np.array([interp(*new_point.T) for interp in
/usr/local/lib/python3.9/site-packages/ezyrb/rbf.py:63: in <listcomp>
    return np.array([interp(*new_point.T) for interp in
E   TypeError: iteration over a 0-d array
```

--- a/pydmd/paramdmd.py
+++ b/pydmd/paramdmd.py
@@ -519,6 +519,12 @@ Property not available now, did you call `fit()`?"""
             untested parameters.
         :rtype: numpy.ndarray
         """
+        if self.parameters is None or len(self.parameters) == 0:
+            raise ValueError(
+                """
+Unknown parameters not found. Did you set `ParametricDMD.parameters`?"""
+            )
+
         if forecasted_modal_coefficients.shape[1] != len(self.dmd_timesteps):
             raise ValueError(
                 "Invalid number of time instants provided: "

--- a/tests/test_paramdmd.py
+++ b/tests/test_paramdmd.py
@@ -523,3 +523,12 @@ def test_interpolated_modal_coefficients_reshape_monolithic():
     )
     np.testing.assert_allclose(p.interpolated_modal_coefficients[1,3], interpolated_modal_coefficients[:,1,3])
     np.testing.assert_allclose(p.interpolated_modal_coefficients[2,0], interpolated_modal_coefficients[:,2,0])
+
+def test_no_parameters_error():
+    dmds = DMD(svd_rank=-1)
+    p = ParametricDMD(dmds, POD(rank=5), RBF(), light=True)
+    p.fit(training_data, params)
+    p.dmd_time['tend'] += 100
+
+    with raises(ValueError, match='Unknown parameters not found. Did you set `ParametricDMD.parameters`?'):
+        p.reconstructed_data

